packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -5078,19 +5079,6 @@ addEventListener('fetch', event => {});`
 					DATA_BLOB_ONE: "./some-data-blob.bin",
 					DATA_BLOB_TWO: "./more-data-blob.bin",
 				},
-				logfwdr: {
-					schema: "./message.capnp.compiled",
-					bindings: [
-						{
-							name: "httplogs",
-							destination: "httplogs",
-						},
-						{
-							name: "trace",
-							destination: "trace",
-						},
-					],
-				},
```

Comment:
Is deleting this section necessary?

Reply:
I can add another mock into this test on the call-capnp function to bring it back, if you'd like